src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copiedState = { ...state };
+  const result = [];
+
+  for (const act of actions) {
+    switch (true) {
+      case (act.type === 'addProperties') :
```

Comment:
will be more informative write like this:
`swtich(act.type) {
  case 'addProperties'
}`